vizro-core/examples/_chart-gallery/pages/part_to_whole.py

```diff
@@ -174,5 +174,34 @@
     ],
 )
 
+funnel = vm.Page(
+    title="Funnel",
+    path="part-to-whole/funnel",
+    layout=vm.Layout(grid=PAGE_GRID),
+    components=[
+        vm.Card(
+            text="""
+
+            #### What is a funnel chart?
+
+            A funnel area chart is a type of data visualization that represents stages in a process, with the size of
+            each area proportional to its value. The chart typically narrows as it progresses, visually depicting the
+            reduction in numbers through each stage. One axis represents the stages of the process, while the other axis
+            indicates the values or quantities at each stage.
+
+            &nbsp;
+
+            #### When should I use it?
+
+            Use a funnel area chart to help your audience understand and compare the progression of data through
+            different stages of a process. This chart type is particularly effective for visualizing conversion rates,
+            sales processes, or any sequential data where you want to highlight drop-offs or reductions between stages.
+        """
+        ),
+        vm.Graph(figure=px.funnel_area(funnel_data, names="Stage", values="Value")),
```

Comment:
Is it possible to import the `funnel_data` from `pages/examples/funnel.py`? If we do the same for all the other examples, we can avoid code duplication and have a single source of truth.

Reply:
The single-source of truth are the ones in `_page_utils.py`, so we always import from there 👍  The ones in `pages/examples/..` are code examples that need to work stand-alone as well, which is why it's difficult to remove the duplication there e.g. the gapminder data set is used in several of these examples py. files. We can't just import the gapminder data set from one of the examples files, because then the other examples don't work as stand-alone anymore. These are the code snippets displayed next to the chart, so we consider them as separate examples :)